MonoGame.Framework/Properties/AssemblyInfo.TVOS.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Resources;
+
+// Specific platform information about an assembly is controlled through the following
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+
+[assembly: AssemblyTitle("MonoGame for TVOS")]
```

Comment:
The correct name is tvOS.  Capitalization is important when it comes to trademarked names.